### ISSUE TITLE
Build process leaves files in root directory: PersistRwAttrList.csv, merged_target_types.xml

### ISSUE THREAD

**Issue by rlippert** [open-power/op-build#1369]
Similar to issue #1093 op-build process now seems to be accidentally creating these 2 files in the buildroot root directory after a build:

```
$ git status
HEAD detached at FETCH_HEAD
Untracked files:
  (use "git add <file>..." to include in what will be committed)

        PersistRwAttrList.csv
        merged_target_types.xml
```

**Comment by williamspatrick** [comment]
open-power/hostboot#111 seems to be addressing one of these.

**Comment by stewart-ibm** [comment]
Fixed in e923c1b4f5cf6da83731ffac2e2d006a3a2351fa.
Closing.
